ui/app/templates/components/auth-info.hbs

```diff
@@ -7,15 +7,12 @@
       <ul class="menu-list">
         {{#if canExpire}}
           <li class="action">
-            <AlertBanner 
-              @type="warning"
-              @message="We've stopped auto-renewing your token due to inactivity.
-                It will expire in {{moment-from-now auth.tokenExpirationDate interval=1000 hideSuffix=true}}.
-                on {{moment-format auth.tokenExpirationDate 'MMMM Do YYYY, h:mm:ss a'}}"
-            />
+            <AlertBanner @type="warning" @message="We've stopped auto-renewing your token due to inactivity.
+                It will expire in {{date-from-now auth.tokenExpirationDate interval=1000 hideSuffix=true}}.
+                on {{date-format auth.tokenExpirationDate 'MMMM Do YYYY, h:mm:ss a'}}" />
           </li>
         {{/if}}
-        {{#if (is-before (now interval=1000) auth.tokenExpirationDate)}}
+        {{#if (and auth.allowExpiration (is-before (now interval=1000) auth.tokenExpirationDate))}}
```

Comment:
Why was the `auth.allowExpiration` added? I think the `canExpire` above checks that so we shouldn't need to add it here.

Reply:
Oh, great catch. I didn't notice `canExpire`. I added it to follow the pattern that I initially set in `cluster.hbs` but you're right -- we don't need this here!